pkgs/development/r-modules/default.nix

```diff
@@ -513,6 +513,7 @@ let
     textshaping = [ pkgs.pkg-config ];
     ragg = [ pkgs.pkg-config ];
     qqconf = [ pkgs.pkg-config ];
+    vapour = with pkgs; [ pkg-config proj gdal ];
```

Comment:
```suggestion
    vapour = with pkgs; [ pkg-config proj.dev gdal ];
```

Reply:
And I guess proj and gdal should go in BuildInputs, pkg-config can stay in native. (see comment from @jbedo)

Reply:
Done, many thanks!